### PR TITLE
A new extension to format datetime.

### PR DESCRIPTION
**Pull request by xinyu-io**
## Description

Convert timestamps and datetime strings into various formats.

## Screencast
form

result

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `media` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @yangxyo 👋 

First, you have another extension in your PR, could you please remove this 😄 

Just out of curiosity, what's the use case of this extension?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by yangxyo** [comment]
I just use it to get the current time when I write my blog. 

2022-11-11 11:11:11

File: extensions/datetime-format-converter/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  type ActionItem = {
+    item: {
+      content: string;
+    };
+  };
+
+  function Actions({ item }: ActionItem) {
+    return (
+      <ActionPanel>
+        <Action.CopyToClipboard content={item.content} />
```
Comment (pernielsentikaer) [inline extensions/datetime-format-converter/src/index.tsx:53]:
```suggestion
        <Action.CopyToClipboard content={item.content} />
        <Action.Paste content={item.content} />
```
Paste directly could be nice

File: extensions/datetime-format-converter/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "datetime-format-converter",
+  "title": "Datetime Format Converter",
```
Comment (pernielsentikaer) [inline extensions/datetime-format-converter/package.json:4]:
```suggestion
  "name": "date-format-converter",
  "title": "Date Format Converter",
```

File: extensions/datetime-format-converter/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "Convert timestamps and datetime strings into various formats.",
+  "icon": "command-icon.png",
+  "author": "yangxy",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "ts",
```
Comment (pernielsentikaer) [inline extensions/datetime-format-converter/package.json:12]:
```suggestion
      "title": "Format Date",
```

File: extensions/datetime-format-converter/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      "subtitle": "Datetime Format",
```
Comment (pernielsentikaer) [inline extensions/datetime-format-converter/package.json:13]:
```suggestion
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @yangxyo 👋 

If you don't know how to do it or have no clue about it, then it's quite hard to guide you to do it. 

Could be better just to start all over – move your extension out of the directory, and then checkout `raycast/extensions` again and push it again – if it's a new PR it's also okay

I left some feedback for you to address 😄

**Comment by sxn** [comment]
hi @yangxyo - hope you don't mind, I've taken the liberty to delete the extra extension & implementing @pernielsentikaer 's changes. 😸

**Comment by yangxyo** [comment]
> hi @yangxyo - hope you don't mind, I've taken the liberty to delete the extra extension & implementing @pernielsentikaer 's changes for you. 😸 Feel free to revert them if you disagree with them. 😉 

Thx very much.

**Pull request merged**